tests/product/base_product_case.py

```diff
@@ -402,7 +424,7 @@ def assert_started(self, process_per_host):
         return process_per_host
 
     def escape_for_regex(self, expected):
-        expected = expected.replace('{rpm}', PRESTO_RPM)
+        expected = expected.replace('{rpm}', self.presto_rpm_filename)
```

Comment:
A few more comment (some more related to your previous patch)
1. is the basename, just the rpm name without .rpm at the end?  In that case, it should be changed also.  Also, if that's true then we don't actually need a regex here.  For most cases we should just replace the {rpm}/ {rpm_basename} and do a straight up equals or EqualsIgnoringOrder comparison.  
2. There's already a method  re.escape(). 
3. this method would be a problem if you want to replace rpm names, but have actual regexes in the line (something that is using the [] or () in a regex-y way.